apps/modernization-ui/src/pages/advancedSearch/AdvancedSearch.scss

```diff
@@ -49,6 +49,7 @@ button {
     border-bottom: 1px solid #e7e9e9;
     .add-patient-button {
         height: fit-content;
+        margin: 0 !important;
```

Comment:
question:  Should this style be in the Advanced Search styles?  It's called `add-patient-button` but it seems more like a general button style.  Should it be in the `shared.scss` file?

Reply:
Yeah you are right we can move it to the shared.scss because it is a general button